core_lib/helpers/string.py

```diff
@@ -1,6 +1,21 @@
+import re
+from re import split
+
+
+def _is_camel_case(string: str) -> bool:
+    return string != string.lower() and string != string.upper() and "_" not in string and " " not in string
+
+
 def snake_to_camel(snake_str) -> str:
     return ''.join(x.title() for x in snake_str.split('_'))
 
 
 def camel_to_snake(s) -> str:
     return ''.join(['_' + c.lower() if c.isupper() else c for c in s]).lstrip('_')
+
+
+def any_to_camel(string: str) -> str:
```

Comment:
yes, because what is happening is if the user inputs a camel case `UpdateSellerData` or `updaateSellerData` then the function converts it to `Updatesellerdata` so just kept a function to check for camel case and do the formatting

Reply:
I had tried the stackoverflow answers and the stringcase implementation too, but the current implementation works well for most of the cases, it works well where the others are failing